packages/common/src/utils/errors.ts

```diff
@@ -0,0 +1,27 @@
+export class BadRequest extends Error {
```

Comment:
never used this pattern, but I can see its usefulness in that other file's "switch"  💡